lib/active_model/serializer/adapter/flatten_json.rb

```diff
@@ -1,19 +1,13 @@
-module ActiveModel
-  class Serializer
-    class Adapter
-      class FlattenJson < Json
-        def serializable_hash(options = {})
-          super
-          @result
-        end
+class ActiveModel::Serializer::Adapter::FlattenJson < ActiveModel::Serializer::Adapter::Json
```

Comment:
even for classes as namespaces? link plz?  (besides that it doesn't work in this case, I think it's wrong)